docs/_static/js/script.js

```diff
@@ -28,7 +28,7 @@ $(function() {
             '#build-threadless-version-not-recommended',
             '#build-mpi-version',
             '#build-gpu-version',
-            '#build-cuda-version-experimental',
+            '#build-cuda-version',
```

Comment:
This anchor `#build-cuda-version-experimental` does not exist, and I think that's why the CUDA instructions are the only ones not collapsed on https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html.

<img width="766" alt="image" src="https://github.com/microsoft/LightGBM/assets/7608904/a3bb05fa-27fa-44e8-a1ac-cc8f07570e3e">